pkg/etcdutil/layout.go

```diff
@@ -64,12 +63,12 @@ func TaskMasterPath(appName string, taskID uint64) string {
 	return path.Join("/", appName, TasksDir, strconv.FormatUint(taskID, 10), TaskMaster)
 }
 
-func MetaPath(linkType, appName string, taskID uint64) string {
+func MetaPath(appName string, taskID uint64) string {
 	return path.Join("/",
 		appName,
 		TasksDir,
 		strconv.FormatUint(taskID, 10),
-		linkType)
+		"meta")
```

Comment:
I think we agree on the same thing. And that's what previous code failed on.

What you're worried about is the link type. I think that's undefined yet. So can you think of how to define it?